### PR TITLE
Support non root permissions for ML docker 

### PR DESCRIPTION
**Pull request by lperryp**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/35834

## Description
Adding support for running nltk and matplotlib packages without root permissions

### CODE REVIEW

**Comment by lperryp** [comment]
@glicht  - can you please check this? we're having issues when `docker.run.internal.asuser==true`, and python packages can't access their resources on the docker, this solution seems like the right approach?

**Review by erezh31 — APPROVED** [review]

File: docker/ml/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,16 @@ RUN apt-get update \
...
+
+USER ml
+
 RUN python -m spacy download en_core_web_sm
-RUN python -c "import nltk; nltk.download('stopwords'); nltk.download('punkt')"
-COPY encrypted_model.b ./var
-COPY oob_evaluation.txt ./var
+RUN python -c "import nltk; nltk.download('stopwords', download_dir='/ml/nltk_data'); nltk.download('punkt',download_dir='/ml/nltk_data')"
+ENV NLTK_DATA='/ml/nltk_data'
+ENV MPLCONFIGDIR='ml/matplotlib'
```
Comment (glicht) [inline docker/ml/Dockerfile:25]: Don't you need a `/`
```suggestion
ENV MPLCONFIGDIR='/ml/matplotlib'
```

Comment (lperryp) [reply docker/ml/Dockerfile:25]: thanks

**Review by glicht — CHANGES_REQUESTED** [review]
The part about adding the `ml` user can be removed. We run with a dedicated user. I would remove the following:
```
WORKDIR /ml
RUN useradd -m ml
RUN chown -R ml:ml /ml

USER ml
```

**after** the download is done do the following:

```
RUN chown -R demisto:demisto /ml && chmod -R 775 /ml
```

**Review by glicht — APPROVED** [review]

**Pull request merged**